src/MediaServiceProvider.php

```diff
@@ -28,7 +31,18 @@ public function register(): void
         $this->app->bind(BulkCreateMediaRequestContract::class, BulkCreateMediaRequest::class);
         $this->app->bind(SearchMediaRequestContract::class, SearchMediaRequest::class);
         $this->app->bind(DeleteMediaRequestContract::class, DeleteMediaRequest::class);
-
         $this->app->bind(MediaServiceContract::class, MediaService::class);
+
+        $this->includeEnvFile();
+    }
+
+    protected function includeEnvFile(): void
```

Comment:
not sure that we need it, please just commit `.env.testing` file

Reply:
Done.